packages/techdocs-cli/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
+  overrides: [
+    {
+      files: ['**/*.ts?(x)'],
+      rules: {
+        'no-restricted-imports': 0,
```

Comment:
This was being flagged for things like `import path from 'path`` etc. Just trying to keep the number of changes to a minimum.